src/DistributedFactorGraphs.jl

```diff
@@ -291,7 +291,7 @@ function __init__()
     end
 
     # if get(ENV, "DFG_USE_CGDFG", "") == "true"
```

Comment:
Good point, think we want the if statement as well?

Reply:
that will close #555 too, we can add whenever is good.  Just helping review code while I chew on other stuff.  You should hide and focus on big items for this week.  We can catch up after.

I will add and let you merge when you get a moment.

Reply:
ha, thanks!